lib/commandFactory.js

```diff
@@ -74,9 +74,13 @@ class CommandFactory extends BaseFactory {
                 const patch = parseInt(latestPatch.slice(1), 10) + 1;
 
                 config.version = `${latestMajor}${latestMinor}.${patch}`;
+                latest.latestVersion = false;
+
+                await latest.update();
```

Comment:
does the function on line 64 need `async` since you're using `await` here?

Reply:
yep, hence the build failure